File: CodinGame.Application/client/src/models/mars-lander/Map.ts
import {MapElement} from "@/models/mars-lander/MapElement";

export class Map {
  public Name!: string;
  public SurfaceElements!: MapElement[];
  public InitialX!: number;
  public InitialY!: number;
  public InitialHorizontalSpeed!: number;
  public InitialVerticalSpeed!: number;
  public InitialFuel!: number;
  public InitialRotation!: number;
  public InitialPower!: number;
}

File: CodinGame/MarsLander/Models/Dtos/Map.cs
using System.Collections;
using System.Collections.Generic;

namespace CodinGame.MarsLander.Models.Dtos
{
    public class Map
    {
        public string Name { get; set; }
        public IEnumerable<SurfaceElement> SurfaceElements { get; set; }
        public int InitialX { get; set; }
        public int InitialY { get; set; }
        public int InitialHorizontalSpeed { get; set; }
        public int InitialVerticalSpeed { get; set; }
        public int InitialFuel { get; set; }
        public int InitialRotation { get; set; }
        public int InitialPower { get; set; }
    }
}